### PR TITLE
Update setup.py dependencies.

### PR DESCRIPTION
**Pull request by pgzmnk**
Set install dependencies to:

``` python
install_requires=['apache-airflow>=1.10',
                      'great-expectations>0.13.7',
                      'pybigquery>0.3.0',
                      'SQLAlchemy>=1.3.18,<1.4']
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,9 +16,11 @@
     long_description_content_type='text/markdown',
     url="https://github.com/great-expectations/airflow-provider-great-expectations",
     install_requires=['apache-airflow>=1.10',
-                      'great-expectations<=0.13.7',
-                      'pybigquery==0.5.0'],
-    packages=['great_expectations_provider'],
+                      'great-expectations==0.13.14',
```
Comment (talagluck) [inline setup.py:19]:
```suggestion
                      'great-expectation>=0.13.9',
```

**Review by talagluck — APPROVED** [review]
Thanks for the contribution! I updated the `install_requires` to match the `requirements.txt` - I believe any version of 0.13.8 should be fine, but that is 0.13.9 to be safe.

**Comment by talagluck** [comment]
@pgzmnk, once you sign our CLA I can get this merged - please let me know if you have any questions or any trouble with that. Thanks!

**Comment by talagluck** [comment]
@cla-bot check

**Comment by talagluck** [comment]
@cla-bot check

**Comment by talagluck** [comment]
@cla-bot check

**Comment by pgzmnk** [comment]
@cla-bot check

**Review by talagluck — APPROVED** [review]
LGTM

**Pull request merged**